blackbox/docs/admin/system-information.rst

```diff
@@ -613,12 +613,12 @@ in subsequent versions. All ``LONG`` columns always return ``0``.
 |                                       | established via   |                   |
 |                                       | HTTP              |                   |
 +---------------------------------------+-------------------+-------------------+
-| ``http[currently_open]``              | The currently     | ``LONG``          |
+| ``http[open]``                        | The currently     | ``LONG``          |
```

Comment:
+1 for "open" and "total"

`open` however should be single-quoted, same with `total`